scripts/init_sovrin_raet_keep

```diff
@@ -4,7 +4,7 @@ import argparse
 import os.path
 
 from plenum.common.raet import initLocalKeep
-from plenum.common.types import CLIENT_STACK_SUFFIX
+from plenum.common.constants import CLIENT_STACK_SUFFIX
```

Comment:
As I said in another review, I think the wisdom of a module named "constants" is iffy. May or may not be the right choice.